projects/plugins/social/src/js/components/icon-text-section/index.js

```diff
@@ -0,0 +1,16 @@
+import { Container, Text } from '@automattic/jetpack-components';
+import styles from './styles.module.scss';
+
+const IconTextSection = ( { icon, title, children } ) => (
+	<Container horizontalSpacing={ 7 } horizontalGap={ 3 }>
```

Comment:
What do you think about moving the `Container` wrapper to the `SupportSection` component and renaming this one to `IconText`? That way this component just handles the icon and text and doesn't concern itself with any layout formatting.

Reply:
Good idea! I did this.